documentation/sphinx/source/special-keys.rst

```diff
@@ -267,3 +267,18 @@ clients can connect FoundationDB transactions to outside events.
 .. [#conflicting_keys] In practice, the transaction probably committed successfully. However, if you're running multiple resolvers then it's possible for a transaction to cause another to abort even if it doesn't commit successfully.
 .. [#max_read_transaction_life_versions] The number 5000000 comes from the server knob MAX_READ_TRANSACTION_LIFE_VERSIONS
 .. [#special_key_space_enable_writes] Enabling this option enables other transaction options, such as ``ACCESS_SYSTEM_KEYS``. This may change in the future.
+
+Versioning
+=================
+
+For how FDB clients deal with versioning, see :ref:`api-versions`.
+The special key space deals with the versioning by using the ``API_VERSION`` that passed to initilialize the client.
+The framework will ingore all modules added at the version larger than the api version set by the client.
+For example, if a module is added in version 7.0 but the client set the api version as 6.3, then the module it not available to the client.
+According to the definition of this contract, anytime you want to remove or update existing modules, you need to keep the old implementation to let the client get the same behavior when it's using the old api version.
```

Comment:
Done. Thanks.

Reply:
Apologies, I didn't specify which section I was referring to before. I think the section below this comment was fine before, it was listed as several independent paragraphs. The section above this comment is what I was concerned about, since it consists of several sentences separated by single line breaks.